src/Sample.Shared.Views/Content/Cartography/DynamicMap_MoveSearchPage.xaml

```diff
@@ -43,33 +44,13 @@
 					Background="Black">
 			<TextBlock Text="{Binding ViewPortCoordinates}"
 					   Foreground="White"
-					   TextWrapping="Wrap"
-					    />
+					   TextWrapping="Wrap" />
 
 			<TextBlock Text="{Binding Pushpins, Converter={StaticResource FromMoveSearchPushpinListToString}}"
 					   TextWrapping="Wrap"
-					   Foreground="White"
-					   />
+					   Foreground="White" />
 		</StackPanel>
 
-		<!-- Dynamic map -->
```

Comment:
Is it normal to remove the map from this map sample?

Reply:
I'm not sure, I'll check